redis_connection_verifier.sh

```diff
@@ -18,9 +18,17 @@
 # 2. Execute the script by running:
 #    ./redis_connection_verifier.sh
 
-REDIS_HOST="192.168.2.1" # Change this to where your Redis server is running
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    echo "MacOS"
+    if ! command -v gtimeout &> /dev/null; then
+        echo "Installing coreutils"
+        brew install coreutils
+    fi
+fi
+
+REDIS_HOST="localhost" # Change this to where your Redis server is running
```

Comment:
Hmmm, but this script is targeted at students who want to test a Redis connection from an external device, like a second laptop or a Nao robot, because that is where many students struggle. I feel that if we keep it set to `localhost`, they may never try to run this script from another machine. They might say, "Yes, this script works, but my robot still can’t connect to Redis" The main purpose of this script is to allow students to check if their Redis server accepts any inbound traffic without having a robot on hand

Reply:
Additionally, running the `demo_desktop_camera.py` is already a good way to test the connection within localhost

Reply:
Yeah, I am thinking the same, but after last week I wonder if the average student knows which IP to `grep` from `ifconfig`. Giving us more questions as the students do not know the usage.

We could maybe `grep` the absolute IP `10.0.0.x` and add this to the script something along the likes of `ifconfig | grep en1` or `ifconfig | grep wlan0` on MacOS/Linux. 

I think this should give the same behaviour as trying to connect externally, as it does not treat the device as a loopback and the ports thus need to be opened.

Reply:
I really think students should know how to find their IP address. How about adding this to the instructions and leaving `REDIS_HOST` as localhost to provide a local connection for testing by default, along with extra information on how to find an IP if needed? 
```
# 1. If you are testing local reachability, you don't need to change REDIS_HOST.
# If you are testing remote reachability, change REDIS_HOST to the IP address of the Redis server.
    # 1.a How to find your private IP address:
    #     - On macOS:
    #         - ipconfig getifaddr en0
    #     - On Linux:
    #         - hostname -I | awk '{print $1}'
# 2. Make the script executable by running:
...
```